domain/tests/src/Unit/Access/DomainAccessCheckTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
To prevent non-relevant changes from appearing in the PR, simply do not commit to the branch that the PR was opened from.

Simply branch out (i.e. create a new branch) and continue your work there.

To save this PR, use `git rebase -i` (see https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history for a nice explanation) -- basically you need to run `git rebase -i HEAD~n` (where N is the the number of commits on this branch, or the number of commits you want to change -- I counted 5)

Before that, do create a new branch off 8.x-1.x and `git cherry-pick COMMIT_HASH` for the unit test commits and the service injection commit